packages/renderer/src/ffmpeg-flags.ts

```diff
@@ -57,3 +62,89 @@ export const getFfmpegVersion = async (options: {
 	});
 	return parseFfmpegVersion(buildInfo);
 };
+
+export const downloadFfmpeg = async (): Promise<void> => {
+	// implement callback instead
+	function onProgress(downloadedBytes: number, totalBytes: number) {
+		console.log(
+			'Downloading ffmpeg: ',
+			toMegabytes(downloadedBytes) + '/' + toMegabytes(totalBytes)
+		);
+	}
+
+	const os = require('os');
+	const path = require('path');
+	const destinationPath =
+		os.platform() === 'win32'
+			? path.resolve(process.cwd(), 'node_modules/.ffmpeg/ffmpeg.exe')
+			: path.resolve(process.cwd(), 'node_modules/.ffmpeg/ffmpeg');
+	if (!fs.existsSync(path.resolve(process.cwd(), 'node_modules/.ffmpeg'))) {
+		fs.mkdirSync(path.resolve(process.cwd(), 'node_modules/.ffmpeg'));
+	}
+
+	console.log(destinationPath);
+	let url: string;
+
+	if (os.platform() === 'win32') {
+		url =
+			'https://remotion-ffmpeg-binaries.s3.eu-central-1.amazonaws.com/ffmpeg-win-x86.exe';
+	} else if (os.platform() === 'darwin') {
+		url =
+			process.arch === 'arm64'
+				? 'https://remotion-ffmpeg-binaries.s3.eu-central-1.amazonaws.com/ffmpeg-macos-arm64'
+				: 'https://remotion-ffmpeg-binaries.s3.eu-central-1.amazonaws.com/ffmpeg-macos-x86';
+	} else {
+		url =
+			'https://remotion-ffmpeg-binaries.s3.eu-central-1.amazonaws.com/ffmpeg-linux-amd64';
+	}
+
+	try {
+		const totalBytes = await _downloadFile(url, destinationPath, onProgress);
+		onProgress(totalBytes, totalBytes);
+		if (os.platform() !== 'win32') {
+			fs.chmodSync(destinationPath, '755');
+		}
+	} catch (error) {
+		console.log(error);
+	}
+};
+
+const getFfmpegBinaryFromNodeModules = () => {
+	const os = require('os');
+	const isWin = os.platform() === 'win32';
+	const path = require('path');
+	if (isWin) {
```

Comment:
redundancy that can be be eliminated together with all the other instances of this path